airflow/jobs/scheduler_job.py

```diff
@@ -303,10 +303,10 @@ def done(self) -> bool:
         return False
 
     @property
-    def result(self) -> Optional[Tuple[List[dict], int]]:
+    def result(self) -> Optional[int]:
         """
         :return: result of running SchedulerJob.process_file()
-        :rtype: Optional[Tuple[List[dict], int]]
+        :rtype: int or None
```

Comment:
```suggestion
        :rtype: Optional[None]
```

Reply:
Doc typing uses a different syntax to type comments https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists:

> Multiple types in a type field will be linked automatically if separated by the word “or”:
> ```
> :type an_arg: int or None
> :vartype a_var: str or int
> :rtype: float or str
> ```

Reply:
Interesting, TIL 👌

Reply:
This function returns tuple or None. The description in the pydocstring is incorrect.
`int or None` != `Optional[Tuple[int, int]]`